src/routes/RepositoryDetails/Tags/SecurityDetails.tsx

```diff
@@ -92,10 +103,16 @@ export default function SecurityDetails(props: SecurityDetailsProps) {
     return <div>Unsupported</div>;
   }
 
-  if (vulnCount.size === 0) {
+  // API call failed, hide replace skeleton with error indicator
+  if (error) {
+    return <div>Error</div>;
+  }
+
+  if (vulnCount.size) {
     return (
       <Link
         to={getTagDetailPath(props.org, props.repo, props.tag, queryParams)}
+        onClick={() => resetSecurityDetails()}
```

Comment:
Using arrow function inside JSX hurts performance. Could this be changed to `onClick={resetSecurityDetails}`

Reply:
TIL, fixed!